### ISSUE TITLE
X-ray Image Pattern Recognition 

### ISSUE THREAD

**Issue by duFUG** [PredictionIO/PredictionIO#239]
I'd like to recognize certain points on x-rays with the help of pattern recognition. Is there any option to do so in PredictionIO? Could smbd please point me a way. Thank u in advance. Best DU

**Comment by tuxdna** [comment]
Are you talking about X-ray images? How would you describe a pattern that you want to recognize?

**Comment by duFUG** [comment]
Yes I am. Basically I want to detect the cemento-enamel junction and the alveolar bone crest. Here u find a schema of the two points. They do look different in x-rays. Basically I think it's be best to feed the whole xray as bytes and the 2 areas as bytes to PredictionIO and to train a model out of that?!
http://pocketdentistry.com/wp-content/uploads/285/Fundamentals_0229_004.jpg

**Comment by k4hoo** [comment]
i think in theory you can convert the image to some sort of numeric/hexidecimal/encoded data, say for example, array of pixel data.

**Comment by duFUG** [comment]
Thabk you for ur reply. Ok. Let's pretend i hex the binary jpeg file and send it to predictionio. Than i crop two 1x1 px points that represent the points i want predictionio to detect in other x-rays. Do u think that would be a valid approach? And if so, what kind of engine would i take?
Best

**Comment by thesuperzapper** [comment]
Maby try Google's TensorFlow what your describing seems to be suited to
deep learning.

https://www.tensorflow.org
